test/mock-dom/index.js

```diff
@@ -35,6 +35,17 @@ exports.createDom = function(callback) {
         html: "",
         src: [axsSrc, jquerySrc, jqueryExtSrc],
         done: function(errors, window) {
+            // Expose some fields from `window` onto the global namespace
+            //
+            // TODO: Currently we add fields here (see "Node") as we need
+            // them, but this may prove tricky to maintain.
+            global.__proto__ = {
```

Comment:
FWIW: `global.__proto__ = window` had some really nasty side effects, in that I could not re-assign `global.__proto__` later for other tests. Not sure why this is the case, but I'm working on a minimum repro.